ciphey/basemods/Decoders/baconian.py

```diff
@@ -0,0 +1,72 @@
+from typing import Optional, Dict, List
+from ciphey.iface import ParamSpec, Config, Decoder, registry
+
+
+@registry.register_multi((str, str), (bytes, bytes))
+class Baconian(Decoder[str, str]):
+
+    lookup = {'A':'aaaaa', 'B':'aaaab', 'C':'aaaba', 'D':'aaabb', 'E':'aabaa', 
+        'F':'aabab', 'G':'aabba', 'H':'aabbb', 'I':'abaaa', 'J':'abaab', 
+        'K':'ababa', 'L':'ababb', 'M':'abbaa', 'N':'abbab', 'O':'abbba', 
+        'P':'abbbb', 'Q':'baaaa', 'R':'baaab', 'S':'baaba', 'T':'baabb', 
+        'U':'babaa', 'V':'babab', 'W':'babba', 'X':'babbb', 'Y':'bbaaa', 'Z':'bbaab'}
+    
+    lookup_default = {'A':'aaaaa', 'B':'aaaab', 'C':'aaaba', 'D':'aaabb', 'E':'aabaa', 
+        'F':'aabab', 'G':'aabba', 'H':'aabbb', 'I':'abaaa', 'J':'abaaa', 
+        'K':'abaab', 'L':'ababa', 'M':'ababb', 'N':'abbaa', 'O':'abbab', 
+        'P':'abbba', 'Q':'abbbb', 'R':'baaaa', 'S':'baaab', 'T':'baaba', 
+        'U':'baabb', 'V':'baabb', 'W':'babaa', 'X':'babab', 'Y':'babba', 'Z':'babbb'}
+
+
+
+    def decode(self, ctext: str) -> Optional[str]:
+        """
+        Performs Baconian decoding
+        """
+        i = 0
+        
+        default_decoced_string = ""
+        decoced_string = ""
+
+        return_default_string = True
+        while True:
+            if(i < len(ctext)-4): 
+                substr = ctext[i:i + 5] 
+                if(substr[0] != ' '):
+                    decoced_string += list(self.lookup.keys())[list(self.lookup.values()).index(substr)]
```

Comment:
Your tests are failing because 
```
ValueError: b'\xff\x84Y\xb0S' is not in list
```

Is there anyway to check to see if the text is Baconian before you run this? It would be very nice timewise to not run this code if the text is not obviously Baconian :)